vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java

```diff
@@ -120,23 +112,29 @@ public void setBirthDate(LocalDate birthDate) {
             this.birthDate = birthDate;
         }
 
+        public String getAddress() {
+            return address;
+        }
+
+        public void setAddress(String address) {
+            this.address = address;
+        }
+
         @Override
         public String toString() {
             StringBuilder builder = new StringBuilder();
             builder.append(firstName).append(" ").append(lastName);
             if (birthDate != null) {
-                builder.append(", born on ").append(birthDate);
+                builder.append("born on ").append(birthDate).append(", ");
```

Comment:
missing a whitespace in the beginning, same for phone below if no birthdate, etc.

Reply:
Done. (Moved firstname and surname to the block. If they exist, than adding a space after there)